### ISSUE TITLE
Issuing read requests on the KNX bus

### ISSUE THREAD

**Issue by unaxboy** [snowdd1/homebridge-knx#32]
Hi all, I can't get homebridge to work.

I have a RPi, installed homebridge with homebridge-knx plugin and running knxd on it connected to a knx ip gateway ([LogicMachine4](http://openrb.com/logicmachine4/)) at 192.168.10.7 with the following opts: 
KNXD_OPTS="-e 1.1.249 -b ip:192.168.10.7"

Homebridge starts, but I don't see it on iPhone. If i delete all knx staff from config.json, it become visible on iPhone and works well.

My log:

```
pi@raspberrypi:~ $ homebridge
*** WARNING *** The program 'node' uses the Apple Bonjour compatibility layer of Avahi.
*** WARNING *** Please fix your application to use the native API of Avahi!
*** WARNING *** For more information see <http://0pointer.de/avahi-compat?s=libdns_sd&e=node>
*** WARNING *** The program 'node' called 'DNSServiceRegister()' which is not supported (or only supported partially) in the Apple Bonjour compatibility layer of Avahi.
*** WARNING *** Please fix your application to use the native API of Avahi!
*** WARNING *** For more information see <http://0pointer.de/avahi-compat?s=libdns_sd&e=node&f=DNSServiceRegister>
[2016-06-29 21:27:50] Loaded plugin: homebridge-knx
[2016-06-29 21:27:51] Registering platform 'homebridge-knx.KNX'
[2016-06-29 21:27:51] ---
[2016-06-29 21:27:51] Loaded config.json with 0 accessories and 1 platforms.
[2016-06-29 21:27:51] ---
[2016-06-29 21:27:51] Loading 1 platforms...
[2016-06-29 21:27:51] [KNX] Initializing KNX platform...
[2016-06-29 21:27:51] [KNX] Fetching KNX devices.
[2016-06-29 21:27:51] [KNX] parsing acc 0 of 1
[2016-06-29 21:27:51] [KNX] push new device Light
[2016-06-29 21:27:51] [KNX] Accessory constructor called
[2016-06-29 21:27:51] [KNX] created Light accessory
[2016-06-29 21:27:51] [KNX] returning 1 accessories
[2016-06-29 21:27:51] [KNX] Initializing platform accessory 'Light'...
[2016-06-29 21:27:51] [KNX] Preparing Services: 1
[2016-06-29 21:27:51] [KNX] Preparing Service: 0 of type Lightbulb
[2016-06-29 21:27:51] [KNX] [Light]:Lightbulb on/off characteristic enabled
[2016-06-29 21:27:51] [KNX] knx registering BOOLEAN 0/3/2
[2016-06-29 21:27:51] [KNX] [Light]:[On]: Issuing read requests on the KNX bus...
[2016-06-29 21:27:51] [KNX] [knxdevice:knxread] preparing knx request for 0/3/2
[2016-06-29 21:27:51] [KNX] [Light]:Lightbulb Brightness characteristic enabled
[2016-06-29 21:27:51] [KNX] knx registering PERCENT 0/4/2
[2016-06-29 21:27:51] [KNX] [Light]:[Brightness]: Issuing read requests on the KNX bus...
[2016-06-29 21:27:51] [KNX] [knxdevice:knxread] preparing knx request for 0/4/2
>>> knxd groupsocketlisten starting <<<
[2016-06-29 21:27:52] Loading 0 accessories...
Scan this code with your HomeKit App on your iOS device to pair with Homebridge:
                       
    ┌────────────┐     
    │ 123-45-678 │     
    └────────────┘     
                       
[2016-06-29 21:27:52] Homebridge is running on port 51826.
Server running at: http://raspberrypi:3000
[2016-06-29 21:27:53] [KNX] [knxdevice:knxread] knx request sent for 0/3/2
[2016-06-29 21:27:53] [KNX] [knxdevice:knxread] knx request sent for 0/4/2
```

My config.json:

```
{
	"bridge": {
		"name": "Homebridge",
		"username": "CC:22:3D:E3:CE:30",
		"port": 51826,
		"pin": "123-45-678"
	},

	"description": "This is an example configuration file for KNX platform shim",
	"hint": "Always paste into jsonlint.com validation page before starting your homebridge, saves a lot of frustration",
	"hint2": "Replace all group addresses by current addresses of your installation, these are arbitrary examples!",
	"hint3": "For valid services and their characteristics have a look at the KNX.md file in folder platforms!",
	"hint99": "Everything that is named description or hint is not used by homebridge but decriptive only, you may change or remove at will",
	"platforms": [{
		"platform": "KNX",
		"name": "KNX",
		"knxd_ip": "127.0.0.1",
		"knxd_port": 6720,
		"knxd_do_not_read_set_groups": true,
		"hint_knxd_do_not_read_set_groups": "true if homebridge must not send read requests to Set addresses",
		"accessories": [{
			"description": "Only generic type knxdevice is supported, all previous knx types have been merged into that.",
			"name": "Light",
			"services": [{
				"type": "Lightbulb",
				"description": "iOS8 Lightbulb type, supports On (Switch) and Brightness",
				"name": "Cabinet light",
				"On": {
					"Set": "0/0/2",
					"Listen": ["0/3/2"]
				},
				"Brightness": {
					"Set": "0/2/2",
					"Listen": ["0/4/2"]
				}
			}]
		}]

	}],

	"accessories": [

	]
}
```

**Comment by snowdd1** [comment]
That's looking like HomeKit doesn't like the accessory for some reason. Could you post http://localhost:51826/accessories from your raspberry? That's an JSON file with the data HomeKit gets if it asks for the accessories of the bridge.

**Comment by unaxboy** [comment]
accessory file:

`{"status":-70401}`

**Comment by snowdd1** [comment]
If that's all, then it's broken somehow.
Mhh, I just wanted to paste mine, but it doesn't show at all. Have to investigate.

Did you already pair the homebridge with your iPhone/Homekit? I am asking because you wrote "...and works well"

**Comment by unaxboy** [comment]
Yes, but with empty config.json

**Comment by snowdd1** [comment]
It might be the knxd coniguration after all: The back end **ip:** is used for multicast adresses (so you usually are not required to specify one), if you want to connect in tunneling mode, use **ipt:192.168.10.7**

**Comment by ctr49** [comment]
did you discover yet why the accessories file displays 

> {"status":-70401}

only? I get the same response but everything seems to be working otherwise (except for sharing, have to test again and will open a separate issue if it persists)

**Comment by unaxboy** [comment]
> It might be the knxd coniguration after all: The back end ip: is used for multicast adresses (so you usually are not required to specify one), if you want to connect in tunneling mode, use **ipt:192.168.10.7**

I tryed **ipt:192.168.10.7**, same response "_Issuing read requests on the KNX bus_"

**Comment by snowdd1** [comment]
*Issuing read requests* is not an error message but a confirmation that the READ telegram is sent to the knxd. One would expect that the devices on the bus answer (if the GAs are readable), and homebridge-knx can update the services/characteristics appropriately.

**Comment by snowdd1** [comment]
Stale conversation.
